guides/templates/guides/guide_detail.html

```diff
@@ -56,4 +56,6 @@ <h2 class="guide-title">{{ guide.title }}</h2>
   <article>
     {{ guide.content }}
   </article>
+  <br>
+  <i class="muted">Want to talk to <strong>{{ guide.author.first_name }}</strong> about this article? Come <a href="https://discord.gg/010z0Kw1A9ql5c1Qe">join us</a>!</i>
```

Comment:
`first_name` is a bit inconveniently named - it's actually the username received from OAuth: https://github.com/Volcyy/django-allauth/blob/master/allauth/socialaccount/adapter.py#L104, and `last_name` would be the the discriminator. Because of this, it's always present.